text/chapter5.md

```diff
@@ -444,12 +449,18 @@ current :: Number
 current = calculateCurrent lightbulb lightbulb -- uncaught mistake
 ```
 
-Note that the constructor of a newtype often has the same name as the newtype itself, but this is not a requirement. For example, unique names are also valid:
+Note that while a newtype can only have a single constructor, and the constructor must be of a single value, a newtype _can_ take any number of type variables. For example, the following newtype would be a valid definition:
+
+```Haskell
+newtype CouldError err a = CouldError (Either err a)
+```
+
+Also note that the constructor of a newtype often has the same name as the newtype itself, but this is not a requirement. For example, unique names are also valid:
 ```haskell
 {{#include ../exercises/chapter5/src/ChapterExamples.purs:Coulomb}}
 ```
 
-In this case, `Coulomb` is the _type constructor_ and `MakeCoulomb` is the _data constructor_. These constructors live in different namespaces, even when the names are identical, such as with the `Volt` example. This is true for all ADTs. Note that although the type constructor and data constructor can have different names, in practice it is idiomatic for them to share the same name. This is the case with `Amp` and `Volt` types above.
+In this case, `Coulomb` is the _type_ and `MakeCoulomb` is the _data constructor_. These live in different namespaces, even when the names are identical, such as with the `Volt` example. This is true for all ADTs. Note that although the type and data constructor can have different names, in practice it is idiomatic for them to share the same name. This is the case with `Amp` and `Volt` types above.
```

Comment:
It's incorrect to call `Coulomb` a "type constructor", right?  I mean if you say `:k Coulomb` it would just say `Type`

Reply:
I think it's _technically_ correct to say that a "type" is a "type constructor of zero arguments", or as [described by wikipedia](https://en.wikipedia.org/wiki/Type_constructor):
> Basic types are considered to be built using nullary type constructors.

Maybe it would be helpful to point that out somewhere in this section.

Reply:
Ah! Thanks for pointing that out!  What would you think about saying 
```suggestion
In this case, `Coulomb` is the _type constructor_ (of zero arguments) and `MakeCoulomb` is the _data constructor_. These constructors live in different namespaces, even when the names are identical, such as with the `Volt` example. This is true for all ADTs. Note that although the type constructor and data constructor can have different names, in practice it is idiomatic for them to share the same name. This is the case with `Amp` and `Volt` types above.
```

Reply:
Looks good